StoreTestWPF.ViewModel/Enums/WindowTitles.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Reflection;
+
+namespace StoreTestWPF.ViewModel.Enums
+{
+    public enum WindowTitles
+    {
+        [Display(Name = "Edit cake")] Edit,
```

Comment:
Attribute should be placed on a separate row.

Reply:
Fixed